sdk/anomalydetector/ai-anomaly-detector/README.md

```diff
@@ -41,15 +45,15 @@ npm install @azure/ai-anomaly-detector
 
 To create a client object to access the Anomaly Detector API, you will need the `endpoint` of your Anomaly Detector resource and a `credential`. The Anomaly Detector client can use either Azure Active Directory credentials or an API key credential to authenticate.
 
-You can find the endpoint for your Anomaly Detector resource either in the [Azure Portal][azure_portal] or by using the [Azure CLI][azure_cli] snippet below:
+You can find the endpoint for your Anomaly Detector resource either in the Azure Portal or by using the Azure CLI snippet below:
 
 ```bash
 az cognitiveservices account show --name <your-resource-name> --resource-group <your-resource-group-name> --query "endpoint"
 ```
 
 #### Using an API Key
 
-Use the [Azure Portal][azure_portal] to browse to your Anomaly Detector resource and retrieve an API key, or use the [Azure CLI][azure_cli] snippet below:
+Use the Azure Portal to browse to your Anomaly Detector resource and retrieve an API key, or use the Azure CLI snippet below:
```

Comment:
Is there any existing doc page we can link to which goes through the steps of finding the API key in the portal?

Reply:
I couldn't find a doc site with the information but added more details on how to get it in the portal